data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,327 @@
+				goto "end"
+			label "new species"
+			`	"What a peculiar task that is," it looks confused and tilts its head, "what to achieve by this you hope? That an incredible amount of dangers come with meeting other species you not know? For a reason invisible borders in space there are. To be careful and consider every last step I advise."`
+			label "end"
+			`	After a short pause it continues talking, "Your presence here our kind accepts, although not very happy we are. Allowed to travel here you are, but conditions there are. Not too far from here another kind just starts growing, great plans for them our leaders have. Do not interrupt them, hostile actions against them we will revenge."`
+			`	With these words the Quarg turns around and leaves. It seems like you are free to use the local facilities now.`
+
+# Maybe add a version of the ships sprite that looks damaged/modded.
+ship "Nimbus" "Nimbus FTL"
+	add attributes
+		"bunks" -25
+	outfits
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+		"Cloud Piercer" 20
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+mission "Incipias: The Derelict"
+	landing
+	invisible
+	to offer
+		has "Incipias: Meet the Quarg: offered"
+	npc assist
+		government "Hicemus"
+		ship "Nimbus FTL" "HIC Veni #000001"
+		system "Hui'uc"
+		to despawn
+			has "incipias: helped the derelict"
+		on assist
+			conversation
+				`When you approach the ship you get hailed and as you accept the hail color gradients start moving over your screen but your speakers stay silent. It seems like communication over these channels is doomed to fail, you continue your boarding maneuver.`
+				scene "scene/incipias message"
+				`	You manage to dock to the disabled ship quite easily. The lock opens and a hallway opens up before you, obviously not built for humans. The walls are strangely curved and the floor has nearly no grip at all. Your view gets caught by the five giant aliens half octopode, half jellyfish that are pulsating in bright colors, similar to the gradients that appeared on your screen earlier. One of them is floating behind a giant mounted gun, the two barrels are pointed right at you. The gunner flashes in an orange light, and you can feel the tension in the room rising.`
+				choice
+					`	(Try to convey that you intend no harm.)`
+						goto peace
+					`	(Drop the repair equipment and run back into your ship.)`
+						goto flee
+					`	(Try to make a non-lethal shot at the one behind the turret and run!)`
+				`	You hit the creature manning the turret, causing it to explode with a strident sound. Before you can enjoy your short-lived victory, the other turret aims your way. Turning your head you see the gunner flashing in a deep red, before shooting you down, avenging their fallen comrade.`
```

Comment:
Where did the second turret come from?

Reply:
it got lost when rephrasing 😆 , will fix that